core/ui/Actions/new-journal.tid

```diff
@@ -2,15 +2,15 @@ title: $:/core/ui/Actions/new-journal
 tags: $:/tags/Actions
 description: create a new journal tiddler
 
-\define get-tags() $(textFieldTags)$ $(tagsFieldTags)$
 \whitespace trim
-<$vars journalTitleTemplate={{$:/config/NewJournal/Title}} textFieldTags={{$:/config/NewJournal/Tags}} tagsFieldTags={{$:/config/NewJournal/Tags!!tags}} journalText={{$:/config/NewJournal/Text}}>
-<$wikify name="journalTitle" text="<$macrocall $name='now' format=<<journalTitleTemplate>>/>">
+\function tf.get-tags() [<textFieldTags>] [<tagsFieldTags>] +[join[]]
```

Comment:
I think tags need to be joined with a space `+[join[ ]]` -- Also we need to make sure that `[[tags with spaces]]` still work